swarm/network/kademlia.go

```diff
@@ -442,6 +452,49 @@ func (k *Kademlia) ListKnown() []*BzzAddr {
 	return res
 }
 
+func (k *Kademlia) Nearest(chunkRef string) (string, error) {
```

Comment:
this should be commented. also please mention that this is to retrieve only the nearest connected peer as it is not obvious from the function name. in any case i would prefer to return an address or peer rather than a string and delegate the responsibility of stringifying to the caller

Reply:
I will remove this from now, I'd rather not have all the Kademlia conversations in this PR.